pilot/cmd/pilot-agent/options/options.go

```diff
@@ -125,6 +125,9 @@ var (
 	enableBootstrapXdsEnv = env.Register("BOOTSTRAP_XDS_AGENT", false,
 		"If set to true, agent retrieves the bootstrap configuration prior to starting Envoy").Get()
 
+	enableWDSEnv = env.Register("WDS", false,
```

Comment:
this's not good for reading, most of people don't what does WDS mean.

Reply:
Expanded.